cosim/black-parrot-example/vcs/Makefile

```diff
@@ -4,14 +4,35 @@ include ../Makefile.design
 # Accelerator Software Settings
 #############################
 DROMAJO_COSIM ?= 0
-SKIP_DRAM ?= -DSKIP_DRAM_TESTING
 BP_NCPUS ?= 1
 
+ZYNQ_PL_DEBUG ?= 0
```

Comment:
Can we add ZYNQ_PL_DEBUG/ZYNQ_PS_DEBUG to the mk/Makefile.* since they'll be used in every example?

Reply:
moved to those Makefiles